perun-core/src/main/java/cz/metacentrum/perun/core/implApi/AuthzResolverImplApi.java

```diff
@@ -568,4 +570,67 @@
 	 * @return list of authorizedGroups
 	 */
 	List<Group> getAdminGroups(Map<String, Integer> mappingOfValues);
+
+	/**
+	 * Get all Vos where the given user has set one of the given roles
+	 * or the given user is a member of an authorized group with such roles.
+	 *
+	 * @param user for who Vos are retrieved
+	 * @param roles for which Vos are retrieved
+	 * @return List of Vos
```

Comment:
Set of objects (there are more of these typos in the file).

Reply:
Fixed